modules/ROOT/pages/_partials/include-manage-proxies.adoc

```diff
@@ -34,10 +49,14 @@ sudo /opt/anypoint/runtimefabric/rtfctl get monitoring-proxy
 The output should match the value expected.
 // end::monitoring-proxy[]
 // tag::http-proxy[]
+. After setting up the proxy, validate the setup by running the `rtfctl validate` command. See xref:install-rtfctl.adoc#validate[rtfctl Validate] documentation.
 
 == Configure an HTTP Proxy for Services in Runtime Fabric
 
-You can configure HTTP and HTTPS proxies so that outbound components can communicate with the control plane. 
+You can configure HTTP and HTTPS proxies so that outbound components can communicate with the control plane.
+
+[NOTE]
+Starting Runtime Fabric version 2.6.11, the following HTTP proxy is needed for Anypoint Monitoring sidecar or agents as well. SOCK5 proxy is not required for Mule apps deployed on Runtime Fabrics with version less than 2.6.22.
```

Comment:
Is it <2.6.11> or <2.6.22> ? Both versions should be the April 9 release version